snowflake-resource-optimization/README.md

```diff
@@ -0,0 +1,72 @@
+# Snowflake Resource Optimization: Setup & Configuration
+
+<p align="center">
+<img src="./images/setup-config-demo.gif" alt="Incident Central demo gif" style="width: 600px;">
+</p>
+
+## Why you need this
+This Retool app extends [Snowflake's Resource Optimization: Setup & Configuration Quickstart Guide](https://quickstarts.snowflake.com/guide/resource_optimization_setup/index.html?index=..%2F..index#0).
+
+The Resource Optimization Quickstart Guides are meant to help customers better monitor and manage their credit consumption.
```

Comment:
Suggestion for making this a bit more in-line with your tone below (where you sound very personable!):

"If you use Snowflake, this Setup & Configuration Quickstart Guide will help you better monitor and manage how quickly you consume your Snowflake credits—and avoid spending more than you need to. For example, the guides provide commands you can run to surface warehouses that don't have auto-resume and auto-suspend configured."

Reply:
Its just the Setup & Configuration quickstart, they have 4 in the Resource Optimization series

Reply:
condensed section to:

This Retool app extends [Snowflake's Resource Optimization: Setup & Configuration Quickstart Guide](https://quickstarts.snowflake.com/guide/resource_optimization_setup/index.html?index=..%2F..index#0). If you use Snowflake, it will help you better monitor and manage your credit consumption, and avoid spending more than you need to.

This Retool app makes it easy to put the Quickstart Guide into practice, by enabling you to take action from right within an easy-to-use UI. Without having to remember or type any commands, you can set auto-suspend set auto-suspend or auto-resume for your warehouses, suspend or drop idle warehouses, disable or drop idle users, etc.